.gitignore

```diff
@@ -9,3 +9,4 @@
 test-output.json
 /visual-snapshots
 *.profraw
+.idea/
```

Comment:
It's a small thing but editor specific files should be ignored in a global gitignore file and not per-repo.

https://docs.github.com/en/get-started/getting-started-with-git/ignoring-files#configuring-ignored-files-for-all-repositories-on-your-computer